projects/packages/ip/.gitattributes

```diff
@@ -0,0 +1,16 @@
+# Files not needed to be distributed in the package.
+.gitattributes    export-ignore
+.github/          export-ignore
```

Comment:
FYI we usually add phpunit.xml.dist and tests/ into the export-ignore section.
I don't think it matters much though.